presto-main/src/main/java/com/facebook/presto/sessionpropertyproviders/NativeWorkerSessionPropertyProvider.java

```diff
@@ -90,6 +91,12 @@ public NativeWorkerSessionPropertyProvider(FeaturesConfig featuresConfig)
                         "Native Execution only. Reduce() function will throw an error if it encounters an array of size greater than this value.",
                         100000,
                         !nativeExecution),
+                integerProperty(
+                        NATIVE_EXPRESSION_MAX_COMPILED_REGEXES,
+                        "Native Execution only. Controls maximum number of compiled regular expression patterns " +
+                                "per regular expression function instance per thread of execution.",
+                        100,
```

Comment:
looks to be standard for all nativeWorkerSessionProperties, so not requesting a change. but why do we define a default value in the session property vs. have the default value be determined by a config property like we do with everything else.

Reply:
I believe because there is no corresponding config property.
Not sure if one must have such.
There is just a corresponding velox query config property, which is per query.